### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by victormc13**
# Latest Features 💡 👨🏽‍💻 

### Project Requirements

For this pull request, I successfully completed the following tasks:

- I added the **missing JSX elements** to the project, ensuring that they seamlessly fit into the existing structure without introducing any unnecessary functionality.
- The main focus of the changes was to align the project's design with Zeplin's specifications. I meticulously ensured that the final design **matched Zeplin's design** in its entirety, encompassing all elements and applying consistent styling.
- To achieve this design coherence, I leveraged the CSS rules provided by Zeplin. By utilizing these rules, I meticulously styled the website to achieve a consistent and harmonious appearance.

These adjustments and additions were made to fulfill the project's requirements, enhancing its functionality and visual appeal while maintaining strict adherence to the provided design guidelines from Zeplin.

### CODE REVIEW

File: src/components/Components.module.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .header {
```
Comment (OLIPLICHE) [inline src/components/Components.module.css:1]: - Hi, can you try to style your App exactly the same as the one provided on Zeplin? This is what is recommended by the project. Pay attention to the header and make it look like the original day.
- [ ] Original design:

Capture1



- [ ] Your project
Capture

File: src/components/Components.module.css [hunk 2]
```diff
@@ -9,3 +9,205 @@
...
+  justify-content: flex-start;
+  align-items: center;
+  flex-shrink: 1; */
+  display: grid;
+  align-items: center;
+  place-items: center;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+
+  /* background-color: red; */
+}
```
Comment (OLIPLICHE) [inline src/components/Components.module.css]: - Can you erase the commented code? because it creates confusion and also makes your code look dirty.

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @victormc13, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- All the functionality is done 👍 
- Added styling 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Components.module.css [hunk 3]
```diff
@@ -1,11 +1,207 @@
...
   align-items: center;
   text-align: center;
-  justify-content: center;
+  justify-content: space-between;
   grid-template-columns: repeat(auto-fit, minmax(320px, 320px));
   border-radius: 4px;
   border-bottom: 1px solid #e8e8e8;
   background-color: #fff;
 }
+
```
Comment (KanzaTahreem) [inline src/components/Components.module.css]: - [ ] You did well in styling the header; however, a little more work is still needed. As suggested, please make sure that the header looks almost identical to the original design ⛏✨
  - _The `navlinks` are positioned on the left side in the provided design, but yours are centered._
  - _The user profile image should be aligned with the book cards, as shown in the given design._
  
Kindly go through the screenshots below for better clarity 💡
  
Your design | Given design
---| ---|
image | image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @victormc13 

Great work on making the changes requested by a previous reviewer 👏! I really appreciate the effort you put in!  **_However, there are few this that still need to be fixed. But you are almost there!_** 💪 🏁

so close


## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Good job with the design
- ✅ Great job with CSS
- ✅ Correct Linters

## Required Changes 🛑

_Please check the comments under the review._

## Optional suggestions ❕

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @victormc13  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Made changes as requested by the previous reviewer :+1: 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**